src/courses/mappers/11.md

```diff
@@ -0,0 +1,54 @@
+---
+order: 11
+next: 12.md
+title: Mapper Preparations - Setting Up
+author: Charles Hu
+---
+
+## Environment Set Up
+
+Node.js (a runtime environment for JavaScript/TypeScript) and Yarn (a package manager for JavaScript/TypeScript) are external utilities which are utilized extensively within this guide. The following section details their installation process.
```

Comment:
it's not actually a runtime for typescript at all lol.  the typescript is transpiled to javascript, and then node runs it.  is it necessary to explain this to the end user idk, but programmers are usually extremely pedantic (or at least i am lol)

Reply:
Addressed.